### DISCUSSION TITLE
how to make embed messages and send the using trigger (await)

### DISCUSSION THREAD

**Discussion by ORGINRISHITH** [Rapptz/discord.py#8099] [Q&A]
pls tell how to make embed and add embed messages

**Comment by rtk-rnjn** [comment]
If you mean to say how to send discord Embeds then here's the example how it is done.
```py
embed = discord.Embed(title="Title of Embed", description="Embed description", color=0xADD8E6)
await ctx.send(embed=embed)
```
---
Embed can be send using [valid python dictionary](https://discord.com/developers/docs/resources/channel#embed-object)
```py
embed = discord.Embed.from_dict({"title": "Title of Embed", "description": "Embed description"})
await message.channel.send(embed=embed)
```
---
You can also add Embed field, Images, Thumbnails, Footer. Everything is documented [here](https://discordpy.readthedocs.io/en/latest/api.html#discord.Embed)
